File: WebClient/src/app/model/status.ts
export class IStatus {
    constructor(
        public state:string,
        public mode:string,
        public pressure:number,
        public station:number,   
        public start:Date,
        public duration:number,
        public scheduleId:number,
        public inputs:string,
        public outputs:string,
        public updatedAt:Date
    ){}
}

File: Device/DeviceController.Data/Status.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DeviceController.Data
{
    public class Status
    {
        public int Id;
        public string State;
        public string Mode;
        public string Station;
        public DateTime Start;
        public int Duration;
        public int ScheduleId;
        public string Inputs;
        public string Outputs;
        public DateTime CreatedAt;
        public DateTime UpdatedAt;
    }
}
